install/lldap-install.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# Co-Author: remz1337
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y gpg
+msg_ok "Installed Dependencies"
+
+msg_info "Installing lldap"
+DISTRO="Debian"
+os=$(less /etc/os-release | grep "^ID=")
+os="${os:3}"
+if [ "$os" == "ubuntu" ]; then
+  DISTRO="Ubuntu"
```

Comment:
https://download.opensuse.org/repositories/home:/Masgalor:/LLDAP/

`DISTRO="xUbuntu"`
This tells me that you didn't fully test this script.

Reply:
Guilty. Thanks for catching and fixing it